public/app/plugins/datasource/tempo/resultTransformer.ts

```diff
@@ -98,6 +111,167 @@ export function transformTraceList(
   return response;
 }
 
+function transformBase64IDToHexString(base64: string) {
```

Comment:
I assume this transform now needs to be duplicated between ts and go right? Let's put a comment here and in the backend code so that if somebody updates one they will know the other part will also need updating.